test/hotspot/jtreg/vmTestbase/nsk/share/native/native_thread.cpp

```diff
@@ -128,7 +128,11 @@ void* THREAD_start(void* t) {
         return NULL;
     }
 #else // !windows & !sun
-    int result = pthread_create(&(thread->id),NULL,procedure,thread);
+    pthread_attr_t attr;
+    pthread_attr_init(&attr);
```

Comment:
You need a corresponding `pthread_attr_destroy` below.

Reply:
Thank you @dholmes-ora. I have made the suggested the changes.